### PR TITLE
feat(upgrade): add error while validations in upgrade

### PR DESCRIPTION
**Pull request by sinhaashish**
the error is thrown and the pic displays the output of `$?` command 

Screenshot 2023-04-12 at 8 39 39 PM

### CODE REVIEW

**Review by tiagolobocastro — APPROVED** [review]

File: k8s/upgrade/src/user_prompt.rs
Comment (niladrih) [inline k8s/upgrade/src/user_prompt.rs:1]: Should use std_err...

```rust
/// Print error on console.
pub fn error(message: &str, data: &str) {
    eprintln!(
        "{} \n {} ",
        Cyan.bold().italic().paint(message),
        Red.bold().italic().paint(data)
    );
}
```

**Review by niladrih — COMMENTED** [review]
Left one comment

File: k8s/upgrade/src/user_prompt.rs
Comment (sinhaashish) [reply k8s/upgrade/src/user_prompt.rs:1]: done

**Comment by sinhaashish** [comment]
bors merge

File: k8s/upgrade/src/user_prompt.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ pub const REBUILD_WARNING: &str =  "\nThe cluster is rebuilding replica of some
 pub const SINGLE_REPLICA_VOLUME_WARNING: &str =  "\nThe list below shows the single replica volumes in cluster.\nThese single replica volumes may not be accessible during upgrade.\nTo skip this validation, please retry  with '--skip-single-replica-volume-validation` flag.";
 
 /// Warning to users before doing an upgrade.
-pub const CORDONED_NODE_WARNING: &str =  "\nBelow are the list of cordoned nodes. Since there are some cordoned nodes, \nthe rebuild will happen on another nodes and the rebuild may get stuck forever. \nTo skip this validation, please retry  with '--skip-cordoned-node-validation` flag.";
+pub const CORDONED_NODE_WARNING: &str =  "\nOne or more nodes in this cluster are in a Mayastor cordoned state.\nThis implies that the storage space of DiskPools on these nodes cannot be utilized for volume replica rebuilds.\nPlease ensure remaining storage nodes have enough available DiskPool space to accommodate volume replica rebuilds,\nthat get triggered during the upgrade process.\nTo skip this validation, please retry  with '--skip-cordoned-node-validation` flag.\nBelow is a list of the Mayastor cordoned nodes:";
```
Comment (avishnu) [inline k8s/upgrade/src/user_prompt.rs:13]: 1. Extra space after 'retry'
2. You can replace 'retry' with 're-run'

**Review by niladrih — APPROVED** [review]
LGTM

**Comment by sinhaashish** [comment]
bors merge

**Pull request merged**